docs/howto/build-run.md

```diff
@@ -152,6 +152,37 @@ Apart from the steps mentioned below, you may find the
 
 [React Native troubleshooting docs]: https://reactnative.dev/docs/troubleshooting
 
+### `tools\postinstall` is not recognized as an internal or external command
+
+On windowsOS, when running `yarn` or `yarn install`, if you see an error like
+this:
+```
+yarn install
+yarn install v1.22.5
+[1/4] Resolving packages...
+success Already up-to-date.
+$ tools/postinstall
+'tools\postinstall' is not recognized as an internal or external command,
+operable program or batch file.
+error Command failed with exit code 1.
+info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
+```
+then, in our `Package.json` :
+```
+$ head package.json
+{
+  "name": "ZulipMobile",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "postinstall": "tools/postinstall",
+```
+
+To fix this, edit that last line so that it reads:
+```
+    "postinstall": "bash tools/postinstall",
```

Comment:
Can we just make this change in the `package.json`, so it works for everyone? I don't see a downside to doing that, but I could be missing something.

Reply:
I thought of that, but I don't know if these changes in package.json will have a different behave on macOS or not.
if you could test this on macOS, or you know it will not affect the behavior on macOS, I will be happy to open a PR with the changes.

Reply:
Yeah, this should have no effect on macOS or Linux.

The script itself already starts with instructions to run it under `bash`:
```
$ head -1 tools/postinstall 
#!/usr/bin/env bash
```
which is always how we want it to run. The issue is just that the Windows command interpreter apparently doesn't understand that instruction, and `yarn` is apparently trying to use the Windows command interpreter to run these lines from `package.json`.

Here's how I put it [earlier in that thread](https://chat.zulip.org/#narrow/stream/48-mobile/topic/Setting.20up.20development.20environment.20for.20zulip-app/near/1102365):
````quote
The extra `bash` should be redundant, and it's a bit ugly, but if it fixes this problem on Windows then it's worth having.
````

Reply:
I see, on it!